multiple-instances-status.js

```diff
@@ -1,10 +1,30 @@
 var events = new (Npm.require('events').EventEmitter)(),
 	collectionName = process.env.MULTIPLE_INSTANCES_COLLECTION_NAME || 'instances',
-	defaultPingInterval = 2000; // 2s
+	defaultPingInterval = (process.env.MULTIPLE_INSTANCES_PING_INTERVAL || 10) * 1000; // default to 10s
 
 var Intances = new Meteor.Collection(collectionName);
```

Comment:
I know this isn't new.  :grin: but should be `instances`  Just noticed :grin:

Reply:
ohh.. a typo.. let me fix it